hs_discover/public/index.html

```diff
@@ -1,17 +1,218 @@
-{% extends "base.html" %}
-{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
-{% block meta_title %}Sample{% endblock %}
+{% extends "newbase.html" %}
+<!--{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}-->
+{% block meta_title %}Discover{% endblock %}
 {% block main %}
-    <input type="hidden" id="static-url" value="{{ STATIC_URL }}">
-    <div class="container" id="discover-main">
-        <div class="row">
-            <div class="col-xs-12">
-                <h2 class="page-title">Success</h2>
-                <div>{{ sample_item }}</div>
-                <div id="app"></div>
+<!-- Google map needs explicit height or may not show up -->
+<!-- vue-date-pick needs font-size override or will inherit an odd oversized appearance from the import of the css -->
+<style type="text/css">
+    /* Always set the map height explicitly to define the size of the div
+     * element that contains the map. */
+    #map {
+        /* a height must be specified to make visible */
+        height: 50%;
+    }
+    .vdpHeadCellContent {
```

Comment:
AddressDjango/global CSS scope mess and tech debt across templates and script includes. Also, bootstrap version conflicts as bootstrap/jQuery are buried deep in the base and will need to be carefully backed out over time. I have the beginngs of a plan for this, but is hidden work that needs to be quantified and justified to business owners.